<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid Walls</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GravityCB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gravity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FuelCB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fuel</string>
    </property>
   </widget>
   <widget class="QSlider" name="gravSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>29</number>
    </property>
    <property name="sliderPosition">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="CollisionsCB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Collisions</string>
    </property>
   </widget>
   <widget class="QSlider" name="fuelSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>225</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
